### ISSUE TITLE
Error: "cat: can't open '/engine.json': No such file or directory"

### ISSUE THREAD

**Issue by bartist** [gilbarbara/codeclimate-stylelint#5]
I'm trying to add StyleLint to Codeclimate as described in [the Installation](https://github.com/gilbarbara/codeclimate-stylelint#installation). However, when running ``codeclimate test stylelint`` I get the following error code:
```
cat: can't open '/engine.json': No such file or directory
error: (JSON::ParserError) 743: unexpected token at ''
```
Is this a problem with one of my setting? or is this unexpected behavior of codeclimate-stylelint?
my stylelint config file is the standard config transformed into a .stylelintrc.yml (in the root folder of the project.

**Comment by hbirkdale** [comment]
I'm having the same problem with Brakeman

**Comment by leonelgalan** [comment]
Why are you testing the engine? You don't have to in order to use it. Apart from #8, _codeclimate-stylelint_ is running for me.

If you are developing for it (forking it, etc.), the process is quiet different.

**Comment by gilbarbara** [comment]
Fixed in d385f65a0e4b22358e45a32ad20987e3a0e755a0
It will be in the next update. @dblandin 😀

**Comment by bartist** [comment]
Thank you! Looking forward to the update!
